packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -4,15 +4,18 @@ import styled from "styled-components"
 import { color, media, space } from "../../helpers"
 import { CloseIcon } from "../../svgs"
 import { ArtsyLogoBlackIcon } from "../../svgs/ArtsyLogoBlackIcon"
+import { usePrevious } from "../../utils/usePrevious"
 import { Box } from "../Box"
 import { Flex } from "../Flex"
 import { Spacer } from "../Spacer"
 import { Serif } from "../Typography"
 
+/**
+ * refreshModalContentKey should change if the modal displays new content and should fade in/fade out with content update
+ */
 interface ModalProps {
   FixedButton?: JSX.Element
-  // refreshModalContent should change if the modal displays new content
-  refreshModalContent?: string
+  refreshModalContentKey?: string
```

Comment:
If this doesn't change and props are passed, does the content update immediately?

Reply:
yes it updates immediately if this key isn't changed.

Reply:
Nice, that's ideal 👍

Reply:
Can a comment be added somewhere describing this?